assets/src/edit-story/components/canvas/editElement.js

```diff
@@ -31,14 +31,11 @@ import {
 } from '../../elements/shared';
 import { useUnits } from '../../units';
 
-// Background color is used to make the edited element more prominent and
-// easier to see.
 const Wrapper = styled.div`
 	${elementWithPosition}
 	${elementWithSize}
 	${elementWithRotation}
 	pointer-events: initial;
-	background-color: ${({ theme }) => theme.colors.whiteout};
```

Comment:
Note: Since this is now moved to text edit only then the whiteout is not added to Media edit mode, however:
1. It's "behind" the media anyway and not visible.
2. It would be visible in case of media with opacity, however, that would interfere with how the image looks like, so IMHO it's not really necessary.

LMK if you think otherwise.

Reply:
could you perhaps add some before/after screenshots in the "user-facing changes" section?

Reply:
Sure, added!

Reply:
Hmm, now looking at it, it looks like the image color is changing anyway when there's opacity -- with the grayout!